.github/workflows/go.yml

```diff
@@ -12,40 +12,36 @@ jobs:
     steps:
 
     - name: Set up Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v5
       with:
         stable: 'false'
```

Comment:
This action now reports that the `stable` field does not exist https://github.com/janos/singleflight/actions/runs/7943839023/job/21688910665#step:2:1. It can be removed.